schism/compiler.ss

```diff
@@ -40,6 +40,7 @@
 
   (define (allocation-pointer) 0)
   (define (word-size) 4)
+  (define (symbol-table-width) 32)
```

Comment:
nit: We should probably have a comment here saying the symbol table size must be a power of 2.